aws-lc-fips-sys/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "aws-lc-fips-sys"
 description = "AWS-LC is a general-purpose cryptographic library maintained by the AWS Cryptography team for AWS and their customers. This is the FIPS validated version of AWS-LC."
-version = "0.12.13"
-links = "aws_lc_fips_0_12_13"
+version = "0.13.0"
```

Comment:
Since this only adds a new header file (which seems unlikely to break backwards compatibility), I think we should make the version `0.12.14`.